Selenium.Axe/Selenium.Axe/HtmlReport.cs

```diff
@@ -0,0 +1,249 @@
+﻿using HtmlAgilityPack;
+using OpenQA.Selenium;
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Web;
+
+namespace Selenium.Axe
+{
+    public static class HtmlReport
+    {
+        public static void CreateReport(this IWebDriver webDriver, string destination)
+        {
+            AxeBuilder axeBuilder = new AxeBuilder(webDriver);
+            webDriver.CreateReport(axeBuilder.Analyze(), destination);
+        }
+
+        public static void CreateReport(this IWebDriver webDriver, IWebElement context, string destination)
+        {
+            AxeBuilder axeBuilder = new AxeBuilder(webDriver);
+            context.CreateReport(axeBuilder.Analyze(context), destination);
+        }
+        public static void CreateReport(this ISearchContext context, AxeResult results, string destination)
```

Comment:
It would be good to run an example of the generated report through an accessibility assessment (a test running it through axe-core would be a good start, but ideally also a fuller assessment with something like [Accessibility Insights](https://accessibilityinsights.io) (I know, I know, I'm biased :)). I haven't run the code and generated a report, but off the top of my head I notice a few issues that an assessment would catch (header level ordering, logical lists that aren't semantic lists, missing alt text on the screenshot...)

Reply:
Let's take this outside the code review, I am not 100% I am following.